File: Src/java/elm/src/jvmTest/java/org/cqframework/cql/elm/serializing/ElmJsonLibraryReaderTest.java
package org.cqframework.cql.elm.serializing;

import org.hl7.elm.r1.Library;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ElmJsonLibraryReaderTest {

    @Test
    void read() {
        var reader = new ElmJsonLibraryReader();
        Library library = reader.read("{\"library\" : { \"type\" : \"Library\"}}");
        assertNotNull(library);
    }
}

File: Src/java/elm/src/jvmTest/kotlin/org/cqframework/cql/elm/serializing/ElmJsonLibraryReaderTest.kt
package org.cqframework.cql.elm.serializing

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class ElmJsonLibraryReaderTest {
    @Test
    fun read() {
        val reader = ElmJsonLibraryReader()
        val library = reader.read("{\"library\" : { \"type\" : \"Library\"}}")
        Assertions.assertNotNull(library)
    }
}
